source/release-notes/1.0.txt

```diff
@@ -12,6 +12,20 @@ Release Notes for mongosync 1.0
    :depth: 1
    :class: singlecol
 
+.. warning:: Past Release Limitations
```

Comment:
(content location)
I think you can omit this one, no? Because the issue only impacts 1.1 - 1.7.1

Reply:
Fixed.